applications/prompt-keda-lsstcam/values-usdfprod-prompt-processing.yaml

```diff
@@ -0,0 +1,118 @@
+      # Block-style mappings can have lines of any length.
+      main: |-
+        - survey: BLOCK-T407  # FBS SV Field Survey
+          pipelines:
+          # TODO: enable ApPipe only when we're ready (resources, alerts, etc.)
+          - ${PROMPT_PROCESSING_DIR}/pipelines/LSSTCam/SingleFrame.yaml
+          - ${PROMPT_PROCESSING_DIR}/pipelines/LSSTCam/Isr.yaml
+        - survey: BLOCK-T427  # Daytime checkout
+          pipelines: ['${PROMPT_PROCESSING_DIR}/pipelines/LSSTCam/Isr-cal.yaml']
+        # Stray light tests, out of focus
+        - survey: BLOCK-T425
+          pipelines: ['${PROMPT_PROCESSING_DIR}/pipelines/LSSTCam/Isr-cal.yaml']
+        - survey: BLOCK-T426
+          pipelines: ['${PROMPT_PROCESSING_DIR}/pipelines/LSSTCam/Isr-cal.yaml']
+        - survey: BLOCK-T428  # Bright star in field
+          pipelines: ['${PROMPT_PROCESSING_DIR}/pipelines/LSSTCam/Isr-cal.yaml']
+        - survey: BLOCK-T429  # Moonlight
+          pipelines: ['${PROMPT_PROCESSING_DIR}/pipelines/LSSTCam/Isr-cal.yaml']
+        # Manual observations during SV blocks?
+        - survey: BLOCK-T364
+          pipelines: []
+        - survey: BLOCK-T365
+          pipelines: []
+        - survey: BLOCK-T366
+          pipelines: []
+        - survey: BLOCK-T367
+          pipelines: []
+        - survey: BLOCK-T368
+          pipelines: []
+        - survey: BLOCK-T369
+          pipelines: []
+        - survey: BLOCK-T373
+          pipelines: []
+        # Miscellaneous scripts, not always images
+        - {survey: "", pipelines: []}
+        # Ignore unknown events
+        - {pipelines: []}
+      preprocessing: |-
+        # TODO: run if and only if ApPipe is a possibility
+        # - survey: BLOCK-T407
+        #   pipelines: ['${PROMPT_PROCESSING_DIR}/pipelines/LSSTCam/Preprocessing.yaml']
+        - {survey: "", pipelines: []}
+        # Don't preprocess anything unknown
+        - {pipelines: []}
+    # TODO: need to adjust this based on observed slew accuracy
+    preloadPadding: 50
+    calibRepo: s3://rubin-summit-users
+
+  s3:
+    imageBucket: rubin-summit
+    endpointUrl: https://sdfembs3.sdf.slac.stanford.edu
+
+  raw_microservice: http://172.24.5.158:8080/presence
+
+  imageNotifications:
+    kafkaClusterAddress: prompt-processing-2-kafka-bootstrap.kafka:9092
+    topic: rubin-summit-notification
+    # TODO: need to adjust this based on observed nextVisit lead time (depends on scheduler) and PP prep time
+    # The shorter this is, the less capacity is wasted on canceled visits
+    imageTimeout: 120
+
+  apdb:
+    config: s3://rubin-summit-users/apdb_config/cassandra/pp_apdb_lsstcam.yaml
+
+  alerts:
+    username: kafka-admin
+    server: usdf-alert-stream-dev.lsst.cloud:9094
+    topic: lsst-alerts
+
+  # TODO: may need to override for debugging
+  logLevel: timer.lsst.activator=DEBUG timer.lsst.daf.butler=DEBUG lsst.diaPipe=VERBOSE lsst.rbClassify=VERBOSE lsst.daf.butler=VERBOSE
```

Comment:
This is the same as the default. Intentional?

Reply:
Yes -- to make sure that any edits wouldn't omit the existing settings.